packages/renderless/src/color-select-panel/utils/color-points.ts

```
@@ -0,0 +1,92 @@
+    })
+  }
+  const getRelativePos = (wrapper: IColorSelectPanelRef<HTMLElement | undefined>, point: IColorPoint) => {
+    const wrapperEl = wrapper.value!
+    const rect = wrapperEl.getBoundingClientRect()
+    return ((point.cursorLeft / rect.width) * 100).toFixed(0)
+  }
+  const setActivePoint = (point: IColorPoint) => {
+    actviePoint.value = point
+  }
+  const getActviePoint = () => actviePoint
+  const toString = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)